### ISSUE TITLE
3 channel of RGB problem

### ISSUE THREAD

**Issue by JeasonUESTC** [tegg89/SRCNN-Tensorflow#38]
I trained the image in RGB format of 3 channels, but the color distortion of the tested image is very serious. Has anyone trained the same and got good results?

**Comment by YoungMagic** [comment]
Have you get it right? I implemented it in pyTorch. Some pixels in image are very bright.

**Comment by JeasonUESTC** [comment]
@YoungMagic I cannot get better results by directly changing it to 3-channel training. I refer to the 3-channel training of others and respectively send them to the network for training, and then calculate loss in combination. The result is normal.

**Comment by YoungMagic** [comment]
@JeasonUESTC  I found that training with 3 channels do not work at all.  To train with only Y channel and bicubic with Cb, Cr channel will get a close image to the author.

**Comment by manojsingla0906** [comment]
can someone tell me where to find the results after training and testing is done? plz help me its important

**Comment by zhaoyucong** [comment]
@manojsingla0906 sample 文件夹中

**Comment by zhaoyucong** [comment]
@YoungMagic  coulde you share your code?
